code/game/mecha/equipment/weapons/mecha_energy.dm

```diff
@@ -113,12 +113,16 @@
 	energy_drain = 40
 	projectiles_per_shot = 1
 	equip_cooldown = 15
-	projectile = /obj/item/projectile/kinetic/mech
+	var/damage = 40
```

Comment:
Do you have a good reason to do this, instead of creating a new projectile? The projectile system exists literally to avoid having variables like this on the object that's using said projectile.